apps/web/lang/de.json

```
@@ -0,0 +1,359 @@
+        "openingHours-1": "Mo–Fr, 5:00–22:00",
+        "openingHours-2": "Sa-So, 6:00–21:00"
+      }
+    },
+    "phone": {
+      "label": "1 234 567 8901",
+      "details": {
+        "openingHours-1": "Mo–Fr, 5:00–22:00",
+        "openingHours-2": "Sa-So, 6:00–21:00"
+      }
+    }
+  },
+  "bottomLinks": {
+    "terms": "AGB",
+    "privacyPolicy": "Datenschutzerklärung",
+  },
+  "socialLabel": "Gehe zu {label}",
+  "failedInfoHeader": "Es tut uns wirklich leid.",
+  "failedInfoMessage": "Ihre Bestellung konnte nicht abgeschlossen werden.",
+  "successInfoHeader": "Vielen Dank!",
+  "successInfoMessage": "Ihre Bestellung wurde erfolgreich abgeschlossen.",
+  "etaDelivery": "Voraussichtliche Lieferdauer",
+  "shippingAddress": "Lieferadresse",
+  "somethingWentWrongImageAlt": "Bild das erklärt das etwas schief gegangen ist",
+  "orderSuccessfulImageAlt": "Bild das erklärt das die Bestellung erfolgreich war",
+  "backToCheckout": "Zurück zur Kasse",
+  "continueShopping": "Einkauf fortsetzen",
+  "orderErrorMessage": "Etwas ging schief. Versuchen Sie es später noch einmal.",
+  "orderNumber": "Auftrags-ID",
+  "productDetails": "Produkt-Details",
+  "technicalData": "Technische Daten",
+  "customerReviews": "Kundenrezensionen",
+  "customerReviewsNone": "Es hat noch niemand eine Bewertung für dieses Produkt abgegeben",
+  "loadingReviews": "Kundenrezensionen werden geladen...",
+  "reviewsCount": "{Anzahl} Bewertungen",
+  "showAllReviews": "Alle Bewertungen anzeigen",
+  "moreItemsOfThisCategory": "Weitere Produkte in dieser Kategorie",
+  "outOfStock": "Nicht auf Lager",
+  "sale": "Sale",
+  "readMore": "Read more",
+  "readLess": "Read less",
+  "noReviews": "No reviews",
+  "loading": "Loading",
+  "gallery": {
+    "count": "{current} of {total}",
+    "thumb": "Go to slide {index}",
+    "prev": "Previous",
+    "next": "Next",
+    "image": "Slide {index}"
+  },
+  "review": {
+    "verifiedPurchase": "Verified purchase"
+  },
+  "account": {
+    "heading": "Mein Konto",
+    "navBarBottomHeading": "Konto",
+    "logout": "Ausloggen",
+    "back": "Zurück",
+    "accountSettings": {
+      "heading": "Kontoeinstellungen",
+      "section": {
+        "personalData": "Persönliche Daten",
+        "billingDetails": "Rechnungsadressen",
+        "shippingDetails": "Versandadressen"
+      },
+      "personalData": {
+        "yourName": "Ihr Name",
+        "firstName": "Vorname",
+        "lastName": "Nachname",
+        "edit": "Bearbeiten",
+        "change": "Ändern",
+        "contactInformation": "Kontaktinformationen",
+        "yourPassword": "Ihr Passwort",
+        "passwordChange": "Passwort ändern",
+        "newPassword": "Neues Passwort",
+        "newPasswordAgain": "Neues Passwort wiederholen",
+        "changePassword": "Passwort ändern",
+        "currentPassword": "Aktuelles Passwort",
+        "passwordHelp": "Verwenden Sie 8 oder mehr Zeichen mit einer Mischung aus Buchstaben, Zahlen und Symbolen."
```

```suggestion
        "passwordHelp": "8 oder mehr Zeichen mit einer Mischung aus Buchstaben, Zahlen und Symbolen"
```
Is there a reason not to make this consistent with the checkout?